RUMM-1845: Add a note to the docs that upload mapping task is not a part of the default build graph (#84)
DataDog/dd-sdk-android-gradle-plugin: docs/upload_mapping_file.md

| `remoteRepositoryUrl`      | The url of the remote repository where the source code was deployed.  If not provided this value will be resolved from your current GIT configuration during the task execution time.                     |
| `checkProjectDependencies` | This property controls if plugin should check if Datadog SDK is included in the dependencies and if it is not:  "none" - ignore, "warn" - log a warning, "fail" - fail the build with an error (default). |

### Integration with CI/CD pipeline

> By default, the upload mapping task is independent of any other task in the build graph. That means that you need to run it manually when mapping needs to be uploaded.

Reviewer: ```suggestion
By default, the upload mapping task is independent from other tasks in the build graph. Run the task manually when you need to upload mapping.
```

> However, if this step should be done in the CI/CD pipeline and it needs to be a part of build graph, you can make upload task to run after mapping file is generated in the following manner:

Reviewer: ```suggestion
If you want to run this task in a CI/CD pipeline, and the task is required as part of the build graph, you can set the upload task to run after the mapping file is generated.

For example:
```
